tests/pyfunc/test_spark.py

```diff
@@ -44,7 +45,10 @@ def test_spark_udf(self):
         pandas_df = self._pandas_df
         spark_df = self.spark.createDataFrame(pandas_df)
         pyfunc_udf = spark_udf(self.spark, self._model_path, result_type="integer")
-        new_df = spark_df.withColumn("prediction", pyfunc_udf(*self._pandas_df.columns))
+        # TODO: The following does not work why?
```

Comment:
Remove TODO's? We should followup with Xiangrui about this, but not really necessary in this PR.

Reply:
kk.